GetItemLink/GetItemLink.csproj

```diff
@@ -48,7 +48,7 @@
 			<HintPath>$(ResonitePath)Resonite_Data\Managed\FrooxEngine.dll</HintPath>
 		</Reference>
 		<Reference Include="System.ValueTuple">
-		  <HintPath>..\..\..\..\..\..\Program Files (x86)\Steam\steamapps\common\Resonite\Resonite_Data\Managed\System.ValueTuple.dll</HintPath>
+		  <HintPath>$(ResonitePath)Resonite_Data\Managed\System.ValueTuple.dll</HintPath>
```

Comment:
it's not that I'm trying to be mean, but I literally had that code in my pr even before launch, just never uploaded.
also I think it's better to use <GamePath> instead of resonite as the team so far have been making things more platform agnostic eg. NeosAssets -> OfficialAssets

Reply:
I'll fix it when I'm home.
If you had the code why not upload it, you had all this time do so but as soon as someone else wanted to do it you showed interest and actually did something.